iothub/device/src/ClientOptions.cs

```diff
@@ -9,7 +9,7 @@ namespace Microsoft.Azure.Devices.Client
     public class ClientOptions
     {
         /// <summary>
-        /// The Digital Twin Model Id associated with the device client instance.
+        /// The Digital Twins model Id associated with the device client instance.
```

Comment:
do you need a plural here - twins?

Reply:
yes, that is the term that we are using when referring to this feature/ service.